src/esp/gfx/Simulator.cpp

```diff
@@ -202,5 +203,111 @@ bool operator!=(const SimulatorConfiguration& a,
   return !(a == b);
 }
 
+// === Physics Simulator Functions ===
+
+const int Simulator::addObject(const int objectLibIndex, const int sceneID) {
+  if (physicsManager_ != nullptr && sceneID >= 0 && sceneID < sceneID_.size()) {
+    // TODO: change implementation to support multi-world and physics worlds to
+    // own reference to a sceneGraph to avoid this.
+    auto& sceneGraph_ = sceneManager_.getSceneGraph(sceneID);
+    auto& drawables = sceneGraph_.getDrawables();
+    return physicsManager_->addObject(objectLibIndex, &drawables);
+  }
+  return ID_UNDEFINED;
+}
+
+// remove object objectID instance in sceneID
+void Simulator::removeObject(const int objectID, const int sceneID) {
+  if (physicsManager_ != nullptr && sceneID >= 0 && sceneID < sceneID_.size()) {
+    physicsManager_->removeObject(objectID);
+  }
+}
+
+// apply forces and torques to objects
+void Simulator::applyTorque(const Magnum::Vector3& tau,
+                            const int objectID,
+                            const int sceneID) {
+  if (physicsManager_ != nullptr && sceneID >= 0 && sceneID < sceneID_.size()) {
+    physicsManager_->removeObject(objectID);
+  }
+}
```

Comment:
:thinking:  This isn't using `tau` in any way. Bad copypaste from above? :)

Reply:
Yes, thanks for catching this. 😂